pkg/apis/v1alpha/routes/routes_handler.go

```diff
@@ -64,6 +63,11 @@ func Register(ws *restful.WebService) {
 		Produces(restful.MIME_JSON))
 }
 
+type Router struct {
```

Comment:
Router struct show be define in dir models,not in handler

Reply:
This only used in handler, and only for parameter parsing purpose